djstripe/views.py

```diff
@@ -49,7 +49,7 @@ def get_context_data(self, *args, **kwargs):
         context['customer'] = customer
         try:
             context['subscription'] = customer.current_subscription
```

Comment:
The `current_subscription` field was dropped during above, shouldn't this now trigger an exception?